server/game/costs/PlayCardResourceCost.ts

```diff
@@ -95,7 +95,7 @@ export class PlayCardResourceCost<TContext extends AbilityContext = AbilityConte
             const amount = this.getAdjustedCost(context);
             context.costs.resources = amount;
 
-            event.context.player.markUsedAdjusters(context.playType, event.context.source, context);
+            event.context.player.markUsedAdjusters(context.playType, event.context.source, context, event.context.target);
```

Comment:
This wasn't needed to fix the issue but I think it was wrong that it was missing!